### PR TITLE
Implement GET endpoint for GenericSensors Interface

### LINKED ISSUE #48: [API] Implement GET endpoint for GenericSensors Interface
### Purpose

Endpoint to retrieve generic sensor data from the database. 

### Description

HTTP request URL: `localhost:3005/api/generic-sensors`  

Some guidelines:
- Next.js automatically create routes by following the directory structure under `pages`. For example, `pages/api/gps` will create the route `localhost:3005/api/gps`. 
- The implementation will be similar to what was done here: https://github.com/UBCSailbot/website/blob/main/pages/api/gps/index.ts
- Make sure to write tests. You can take a look at https://github.com/UBCSailbot/website/pull/19 for inspiration. (Particularly look at the files under the directories `tests/steps` and `tests/features` 

### Resources

- https://nextjs.org/docs/pages/building-your-application/routing/pages-and-layouts

### PR DESCRIPTION
**Pull request by dk1702**
### Description

- Resolves #48 

### Verification

- [ ] 

### Resources

-

### CODE REVIEW

**Review by jahn18 — CHANGES_REQUESTED** [review]
Please also change the api name to `generic-sensors` (add a dash)

File: models/helper/parser.ts [hunk 1]
```diff
@@ -2,6 +2,8 @@ export const decimal2JSON = (v, i, prev) => {
     if (v !== null && typeof v === 'object') {
       if (v.constructor.name === 'Decimal128')
         prev[i] = parseFloat(v.toString());
+      else if (v.constructor.name === 'bigint')
+        prev[i] = v.toString();
```
[the remark this answers is not in the document]
Comment (dk1702) [reply models/helper/parser.ts:6]: done

File: tests/steps/given.ts [hunk 2]
```diff
@@ -237,3 +259,33 @@ Then('the response data matches the LocalPath data in the database', async funct
...
+         let transformedGenericSensors = genericsensors.map((data) => data.toJSON())
+         return transformedGenericSensors;
+    });
+
+    function bigIntConversion(input: bigint): any {
+        if(typeof input == 'bigint') {
+            return input.toString();
+        }
+        return input;
+      }
```
[the remark this answers is not in the document]
Comment (dk1702) [reply tests/steps/given.ts:279]: done

File: tests/features/genericsensors.feature [hunk 3]
```diff
@@ -0,0 +1,9 @@
+@genericsensors
```
Comment (jahn18) [inline tests/features/genericsensors.feature:1]: As discussed on slack, please add a dash to the tag

**Review by jahn18 — APPROVED** [review]

**Pull request merged**